bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -188,6 +188,21 @@ spec:
           containerPort: {{.OVN_NB_PORT}}
         - name: nb-db-raft-port
           containerPort: {{.OVN_NB_RAFT_PORT}}
+        readinessProbe:
+          exec:
+            command:
+            - /bin/bash
+            - -c
+            - |
+              set -xe
+              OVN_NODES_ARRAY=({{.OVN_NODES}}) 
+              for node in "${OVN_NODES_ARRAY[@]}"; do
+                node_ip=$(getent ahostsv4 "${node}" | grep RAW | awk '{print $1}')
```

Comment:
If I'm reading this correctly, it means that if any node goes down, all of the OVN databases will go not Ready. That strikes me as... not good. Maybe we should just make sure there is at least one other node?

Reply:
Indeed, my bad. I got confused with the lifecycle hook and thought it only runs once. Proposition: I'll change it to detecting if the amount of cluster members is higher than the MaxUnavailable, that should make it more precise.